cabby/geo/geo_item.py

```diff
@@ -110,9 +110,14 @@ class RVSSample:
   instructions: str = attr.ib()
   id: int = attr.ib()
   version: float = attr.ib()
+  entity_span: Dict[str, Tuple[int, int]] = attr.ib()
```

Comment:
Can you add some documentation for the values in this class? In particular, what are the str keys here?

Reply:
Added   Key: entity name. Value: tuple of start and end of the span.